opwen_email_server/api/send_outbound_emails.py

```diff
@@ -9,17 +9,15 @@
 
 EMAIL = SendgridEmailSender(key=config.EMAIL_SENDER_KEY)
 
+logger = LogMixin()
 
-class _Sender(LogMixin):
-    def __call__(self, resource_id: str) -> Tuple[str, int]:
-        email = server_datastore.fetch_email(resource_id)
+def send(resource_id: str) -> Tuple[str, int]:
```

Comment:
Please keep the class-style for this functionality since it'll make it easier to move to dependency injection as per https://github.com/ascoderu/opwen-cloudserver/issues/59. Going forward we'll want to refactor these classes to take in all external dependencies (e.g. `SendgridEmailSender`, `server_datastore`, etc.) via the constructor so that it's easier to test the classes by passing in mocks. You can still import the initialized `send = _Sender()` object from other modules and use it just like a function.

Reply:
Alrighty, will make the change.